packages/documentation/components/NavBar.vue

```diff
@@ -86,28 +86,22 @@ export default defineComponent({
 				},
 			],
 			sections: menu.map(
-				(
-					section,
-				): {
-					title: string | null
-					links: Array<{
-						icon: Yoco.Icon
-						title: string
-						path: string
-						isActive: boolean
-					}>
-				} => ({
-					links: section.subsections.map((subsection) => ({
-						icon: subsection.icon,
-						title: subsection.title,
-						path: `/${subsection.path}${
-							subsection.pages.length >= 1 ? `/${subsection.pages[0].path}` : ''
-						}`,
-						isActive:
-							subsection.path === ''
-								? route.value.path === '/'
-								: route.value.path.startsWith(`/${subsection.path}`),
-					})),
+				(section): Kotti.Navbar.Section => ({
+					links: section.subsections.map(
+						(subsection): Kotti.Navbar.SectionLink & { to: string } => ({
```

Comment:
Removal of `to: z.unknown().optional()` made this necessary

Reply:
that's what I was looking for actually, it gives clarity, that this is an usage-specific API